builder/ecs/builder.go

```diff
@@ -202,6 +202,12 @@ func (b *Builder) Run(ctx context.Context, ui packersdk.Ui, hook packersdk.Hook)
 		})
 	}
 
+	if b.config.SkipCreateImage {
+		steps = append(steps, &stepStopBuilder{
+			Message: SkipCreateImage,
+		})
+	}
+
```

Comment:
Instead of halting the build at this step what do you think about not appending the image creation steps if SkipCreateImage is true. Something like the following. 

```go
	if !b.config.SkipCreateImage {
		steps = append(steps,
			&stepCreateAlicloudImage{
				AlicloudImageIgnoreDataDisks: b.config.AlicloudImageIgnoreDataDisks,
				WaitSnapshotReadyTimeout:     b.getSnapshotReadyTimeout(),
			},
			&stepCreateTags{
				Tags: b.config.AlicloudImageTags,
			},
			&stepRegionCopyAlicloudImage{
				AlicloudImageDestinationRegions: b.config.AlicloudImageDestinationRegions,
				AlicloudImageDestinationNames:   b.config.AlicloudImageDestinationNames,
				RegionId:                        b.config.AlicloudRegion,
				WaitCopyingImageReadyTimeout:    b.getCopyingImageReadyTimeout(),
			},
			&stepShareAlicloudImage{
				AlicloudImageShareAccounts:   b.config.AlicloudImageShareAccounts,
				AlicloudImageUNShareAccounts: b.config.AlicloudImageUNShareAccounts,
				RegionId:                     b.config.AlicloudRegion,
			})
	}
```

Reply:
Fixed